packages/js/ai/src/index.ts

```diff
@@ -11,7 +11,6 @@ export {
  * Utils
  */
 export {
-	requestJetpackToken as __experimentalRequestJetpackToken,
```

Comment:
For cleanliness, I deleted this function entirely. Is that safe to do, or, do we need to care about backward-compatibility already? I suppose we could keep this `requestJetpackToken()` for stability and just have it be a thin wrapper over `requestJwt()`? 🤔

Reply:
Good question. Truthfully this is so new, and marked as experimental, that I'd have a hard time envisioning it being a problem to to take it out. That said, I'm not 100%. We could add a deprecated notice and make it a wrapper over `requestJwt()` as you mention.